pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -93,6 +121,21 @@ func readDockercfg(filePath string) (cfg credentialprovider.DockerConfig, err er
 	return
 }
 
+// readDockerConfigJson reads the contents of a .docker/config.json file into a map
+// with server name keys and AuthEntry values
+func readDockerConfigJson(filePath string) (cfg credentialprovider.DockerConfig, err error) {
+	content, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		return
+	}
+	var config credentialprovider.DockerConfigJson
```

Comment:
Pretty sure declaring a struct var allocates the struct. I think the two are equivalent.